### DISCUSSION TITLE
Experimental: DeriSock Caching support

### DISCUSSION THREAD

**Discussion by psollberger** [psollberger/DeriSock#56] [Ideas]
@Tradyn, I thought I try to make your life a little easier and created a simple caching class, based on the work you do currently. At least based on the part I saw :)

You can check out an example usage here: https://github.com/psollberger/DeriSock/blob/DeriSock.Caching/src/DeriSock.DevTools/ScratchPad.cs

When you run the __DeriSock.DevTools__ project with arguments `--scratchpad` this code is executed.

It would be great if you could check out the __DeriSock.Caching__ branch and tell me what you think.

**Comment by InTheta** [comment]


**Reply by psollberger** [reply]
I don't really get what exactly you try to do.
If you create a SignalR WebSocket Server, why the cache in between?
Why not just run a BackgroundService and relay the received notifications in real time?
